Source/Public/Set-VSTeamDefaultProjectCount.ps1

```diff
@@ -0,0 +1,63 @@
+function Set-VSTeamDefaultProjectCount {
+   [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments", "")]
+   [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "Low")]
+   param(
+      [switch] $Force,
+
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [int] $Count
+   )
+   DynamicParam {
+      $dp = New-Object System.Management.Automation.RuntimeDefinedParameterDictionary
+
+      # Only add these options on Windows Machines
+      if (_isOnWindows) {
```

Comment:
aren't you creating duplicate code here. the clear function has the same. Is it possible to group the dp creation?

Reply:
Good point. It is actually in four places now. Let me see about refactoring this.

Reply:
I think the code is much better now.